fission/package.go

```diff
@@ -145,8 +156,12 @@ func updatePackage(client *client.Client, pkg *crd.Package, envName, envNamespac
 	var srcArchiveMetadata, deployArchiveMetadata *fission.Archive
 	needToBuild := false
 
-	if len(envName) > 0 {
+	if len(envName) > 0 && len(srcArchiveName) > 0 {
```

Comment:
Remove `len(srcArchiveName) > 0` as we discussed.

Reply:
makes sense. thanks for pointing this out.